full-moon/src/tokenizer.rs

```diff
@@ -683,12 +683,34 @@ fn parse_hex_number(code: &str) -> IResult<&str, &str> {
     ))(code)
 }
 
+fn parse_no_int_fractional_number(code: &str) -> IResult<&str, &str> {
+    recognize(pair(
+        opt(digit1),
+        pair(
+            pair(tag("."), digit1),
+            opt(pair(
+                pair(
+                    tag_no_case("e"),
+                    opt(tag("-"))
```

Comment:
There should be a trailing comma here. Make sure you run `cargo fmt`, which will automatically format all your code to the standard.

Reply:
I'll remember that for future!